examples/go/oauth2/hello.go

```diff
@@ -24,11 +24,11 @@ import (
 var iconData []byte
 
 var manifest = apps.Manifest{
-	AppID:       "hello-oauth2",
+	AppID:       "example-oauth2",
```

Comment:
Note: Need to re-test, plan to improve to further demonstrate bindings, including embedded.